packages/esm-patient-test-results-app/src/hiv/hiv-care-and-treatment.tsx

```diff
@@ -42,7 +42,12 @@ const HIVCareAndTreatment = () => {
   if (!loading && !errors.length && roots?.length) {
     return (
       <FilterProvider roots={roots}>
-        <div style={{ padding: 0 }}>
+        <Grid
+          style={{
+            padding: 0,
+            width: isTablet ? 'calc(100vw - 48px - 2.6rem)' : 'calc(100vw - 16rem - 48px - 2.6rem)',
```

Comment:
One day we will have [Container Queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries)...